### PR TITLE
Document invariants. Rename plan -> plot.

### PR DESCRIPTION
**Pull request by d-xo**
Describe data format & invariants. Renamed `plan` to `plot` because it was previously being used as both a verb and a noun, making it very hard to describe the invariants in a concise way.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,57 +1,81 @@
...
+<p align="center">
+<i><code>delegatecall</code> based proxy with an enforced delay</i>
+</p>
 
-`ds-pause` is designed to be used as a component in a governance system, to ensure that the governed
-have time to respond to malicious actions. This can help to constrain the power wielded by the
-governors.
+`ds-pause` is designed to be used as a component in a governance system, to give affected parties
+time to respond to decisions. If those affected by governance decisions have e.g. exit or veto
+rights, then the pause can serve as a effective check on governance power.
```
Comment (asymmetric) [inline readme.md]: *an effective*

File: readme.md [hunk 2]
```diff
@@ -1,57 +1,81 @@
...
+- `eta`: first possible (unix) time of execution
 
-A `plan` consists of:
+Each plan has a unique id, defined as `keccack256(abi.encode(usr, fax, eta))`
+
+## Operations
+
+Plans can be manipulated in the following ways:
+
+- **`plot`**: mark a `plan` as executable
```
Comment (asymmetric) [inline readme.md]: maybe "schedule a `plan`"?

File: readme.md [hunk 3]
```diff
@@ -1,57 +1,81 @@
...
+- **`exec`**: execute a `plan`
+- **`drop`**: cancel a `plan`
+
+## Invariants
 
-- `usr`: the address to `delegatecall` into
-- `fax`: the `calldata` to use
-- `eta`: the time from when the `plan` can be executed
+**`plot`**
+- A `plan` can only be plotted if it's `eta` is after `block.timestamp + delay`
```
Comment (asymmetric) [inline readme.md]: "its" 😅

Comment (d-xo) [reply readme.md]: What would I do without you :pray: :grinning:

**Pull request merged**